keel-orca/src/test/kotlin/com/netflix/spinnaker/keel/constraints/CanaryConstraintEvaluatorTests.kt

```diff
@@ -39,18 +46,12 @@ import strikt.assertions.isEqualTo
 import strikt.assertions.isFalse
 import strikt.assertions.isNotNull
 import strikt.assertions.isTrue
+import strikt.assertions.last
+import strikt.mockk.captured
+import strikt.mockk.isCaptured
 
 internal class CanaryConstraintEvaluatorTests : JUnit5Minutests {
 
```

Comment:
oof this looks like it was v complicated to refactor

Reply:
That and `ApplicationServiceTests` were the hardest ones